### ISSUE TITLE
Error encountered: The directory name is invalid.

### ISSUE THREAD

**Issue by SirMidi** [fholger/RocksmithToTab#42]
Hello,

Below is what I get each time I try to convert my Rocksmith files.  I would really appreciate any help that comes my way.  TIA

[1/8] Opening archive songs.psarc ...
Error encountered:
The directory name is invalid.

[2/8] Opening archive bobmthre_m.psarc ...
Error encountered:
The directory name is invalid.

[3/8] Opening archive elvisusp_m.psarc ...
Error encountered:
The directory name is invalid.

etc

**Comment by fholger** [comment]
Which paths did you choose in the program for the Rocksmith directory and the output folder? Specifically, have you tried a different save location?

**Comment by SirMidi** [comment]
C:\Users\XXXXXXX\Downloads\Rocksmith2014\dlc
C:\Users\ XXXXXXX \Desktop\tabs

I have tried several different save folders and nothing changes.

**Comment by SirMidi** [comment]
Here is the actual directory that Rocksmith is in:

C:\Program Files (x86)\Steam\steamapps\common\Rocksmith2014

sorry...any ideas?

**Comment by fholger** [comment]
Not really, I'm afraid. You could try a path outside your user directory, e.g. C:\temp\RocksmithTabs (you might need to create C:\temp first). Other than that, the error doesn't ring a bell, and it's been years since I've touched the code :(

**Comment by andpd11** [comment]
hi ihave the same problem,  i have a file with just the songs i want to convert and i get the same message
